target/linux/ipq40xx/files-6.12/arch/arm/boot/dts/qcom/qcom-ipq4029-insect-common.dtsi

```diff
@@ -385,16 +400,14 @@
 
 &wifi0 {
 	status = "okay";
-	qcom,ath10k-calibration-variant = "Meraki-MR33";
-	nvmem-cells = <&mac_address 2>;
-	nvmem-cell-names = "mac-address";
+	nvmem-cells = <&precal_factory_1000>;
```

Comment:
If I'm not mistaken, the MAC address-related NVMEM cells should stay together with new caldata, ART blobs do not contain the MACs on `insects`-s